darwin/build.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+set -xe
+
+# Note: We could let Xcode pass through the build settings in the environment, but this currently at least breaks host compiling of tools (GenerateSineTable will then be build for target) and maybe has other unwanted side effects ... so we pass individual settings as script arguments.
+
+TARGET_PLATFORM_NAME=$1
+CONFIGURATION=$2
+
+export DEBUG="n"
+if [ "${CONFIGURATION}" = "Debug" ]; then
+  export DEBUG="y"
+fi
+
```

Comment:
This should be ideally handled by the build system. Ideally we don't redefine debug.

Reply:
What do you mean? This just automatically sets `DEBUG` for the make build system depending on whether _Release_ or _Debug_ is set as the build configuration in Xcode. We can also default to one value and ignore the Xcode configuration, if you prefer. I think it just adds a bit of convenience like this.